guide/src/format/markdown.md

```diff
@@ -220,3 +220,14 @@ To enable it, see the [`output.html.curly-quotes`] config option.
 [tables]: https://github.github.com/gfm/#tables-extension-
 [task list extension]: https://github.github.com/gfm/#task-list-items-extension-
 [`output.html.curly-quotes`]: configuration/renderers.md#html-renderer-options
+
+### Heading attributes { #headingattributes .class1 .class2 }
```

Comment:
There probably isn't a need to add a custom ID or class here. If you were intending this for testing purposes, that should probably go in the `test_book`.